tsconfig.base.json

```diff
@@ -14,6 +14,7 @@
     "noImplicitOverride": true,
     "pretty": true,
     "sourceMap": true,
+    "inlineSources": true,
```

Comment:
Do we sort the options alphabetically? A quick look seems currently the options list is sorted in alphabetical order.

Reply:
it looks like it was sorted at one time but isn't anymore. i'll sort it.